### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by zhangkun07** [flow-diffusion/AVDC_experiments#3]
Thank you for your amazing work, I noticed that only the code for the inference phase is provided in this repository and not a copy of the code on how to train the model, could you please provide another copy of the code when the model is trained. Also, I would like to ask if the entire loss function of the model consists of only the two loss functions in subsection 3.1 and 3.3 of the paper. When summing these two loss functions to construct the whole loss function, what are the weight hyperparameters of each loss function?Looking forward to your reply.
![image](https://github.com/user-attachments/assets/724a4965-7715-40b8-8d17-dbe2381ff37e)
![image](https://github.com/user-attachments/assets/a86887b4-82e9-442c-a0e0-15b4ebdcff75)

**Comment by kbkbowo** [comment]
Hello!



Let me clarify our framework a bit. It roughly consists of two stages:

1. Training stage: Training a video diffusion model. 
In this case, only L_MSE (reconstruction loss) is applied to train the model. 
The repo for training video diffusion model: 
https://github.com/flow-diffusion/AVDC

2. Testing stage: Obtain actions from the video predicted with the model trained above.
L_Trans is applied to track the 3D trajectory of the object of interest. The trajectory is later on converted to actions with a heuristic policy. Note that we do not train a deep neural network here. Instead, we run scipy.optimize to find the optimal SE(3) transformation parameters (In specific, there are 6 parameters to decide) within seconds during test time. The usage can be found in https://github.com/flow-diffusion/AVDC_experiments/blob/main/experiment/rigid_transform.py

In short, the two losses are independent of each other. L_MSE is for training the diffusion model and L_Trans is for obtaining object trajectory during test time. 

Please let me know if anything is still unclear!

**Comment by zhangkun07** [comment]
Thank you very much for your reply, this has solved my confusion!
